lib/ansible/modules/packaging/language/pip.py

```diff
@@ -47,62 +39,51 @@
         If the virtualenv does not exist, it will be created before installing
         packages. The optional virtualenv_site_packages, virtualenv_command,
         and virtualenv_python options affect the creation of the virtualenv.
-    required: false
-    default: null
   virtualenv_site_packages:
-    version_added: "1.0"
     description:
       - Whether the virtual environment will inherit packages from the
         global site-packages directory.  Note that if this setting is
         changed on an already existing virtual environment it will not
         have any effect, the environment must be deleted and newly
         created.
-    required: false
+    type: bool
     default: "no"
-    choices: [ "yes", "no" ]
+    version_added: "1.0"
   virtualenv_command:
-    version_added: "1.1"
     description:
       - The command or a pathname to the command to create the virtual
         environment with. For example C(pyvenv), C(virtualenv),
         C(virtualenv2), C(~/bin/virtualenv), C(/usr/local/bin/virtualenv).
-    required: false
     default: virtualenv
+    version_added: "1.1"
   virtualenv_python:
-    version_added: "2.0"
     description:
       - The Python executable used for creating the virtual environment.
         For example C(python3.5), C(python2.7). When not specified, the
         Python version used to run the ansible module is used. This parameter
         should not be used when C(virtualenv_command) is using C(pyvenv) or
         the C(-m venv) module.
-    required: false
-    default: null
+    version_added: "2.0"
   state:
     description:
       - The state of module
       - The 'forcereinstall' option is only available in Ansible 2.1 and above.
-    required: false
+    choices: [ absent, forcereinstall, latest, present ]
```

Comment:
No quotes?

Reply:
In YAML we only tend to quote what requires quoting. That's the best way to make people learn the rules IMO